public/app/core/components/AppChrome/Notification/Notification.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from '@emotion/css';
+import { formatDistanceToNow } from 'date-fns';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+import { selectWarningsAndErrors } from 'app/core/reducers/appNotification';
+import { useSelector } from 'app/types';
+
+export function Notification() {
+  const styles = useStyles2(getStyles);
+  const notifications = useSelector((state) => selectWarningsAndErrors(state.appNotifications));
```

Comment:
I think it's better if we use selectAll for the hackathon, because we might send info notifications from plugins, which then wouldn't show up here.